src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -911,6 +911,235 @@ public virtual CreatedAtRouteResult CreatedAtRoute(string routeName, object rout
         }
 
         /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedResult Accepted()
+        {
+            string uri = null;
+            return new AcceptedResult(uri, null);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="value">The optional content value to format in the entity body; may be null.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedResult Accepted(object value)
+        {
+            string uri = null;
+            return new AcceptedResult(uri, value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="uri">The optional URI with the location at which the status of requested content can be monitored.
+        /// May be null.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedResult Accepted(Uri uri)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            return new AcceptedResult(uri, null);
```

Comment:
We typically use named parameters for `null` and primitives values (e.g. `count: 10`, `validate: false` etc). Other parameters should either match method arguments or be descriptive.